src/main/java/bio/terra/service/filedata/DrsId.java

```diff
@@ -70,37 +69,37 @@ public String toString() {
         .append("version", version)
         .append("snapshotId", snapshotId)
         .append("fsObjectId", fsObjectId)
+        .append("compactId", compactId)
         .toString();
   }
 
   @Override
   public boolean equals(Object o) {
```

Comment:
Are there unit tests for this method? I didn't immediately see any in DrsIdTest, but it does seem like a good candidate for a sanity check in the form of a unit test to make sure it's behaving as expected. 
Possibly some other operations in this class might also benefit from unit tests?

Reply:
I like this idea.

Reply:
For equals?  Ah, that gets exercised when we do the serialization/deserialization tests (e.g. DrsId -> url -> DrsId). Those happen in the DrsIdServiceTest (e.g. testCompactIdDrsIds)